packages/cli/semver-utils/src/isVersionAhead.ts

```diff
@@ -30,6 +30,9 @@ export function isVersionAhead(a: string, b: string): boolean {
             case "release":
             case "post-release-commit":
                 return false;
+            case "alpha":
+            case "beta":
+                return true;
```

Comment:
if you have `0.0.1-rc0beta1` and `0.0.1-rc1` which would this pick with the ahead version ?

Reply:
nvm i see you cant overlap rc and beta